tests/test_ble_notifications.py

```
@@ -61,3 +62,33 @@ def start_notify(
+    manager = notification_manager
+    manager._subscribe(FROMNUM_UUID, lambda _sender, _data: None)
+    manager._subscribe("char-1", lambda _sender, _data: None)
+
+    class _Client:
+        def __init__(self) -> None:
+            self.calls: list[str] = []
+
+        def start_notify(
+            self,
+            characteristic: str,
+            _callback: Any,
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_callback`

(ANN401)